### DISCUSSION TITLE
List is not updated if an existing starred repo is added to a list.

### DISCUSSION THREAD

**Discussion by santoshyadavdev** [community/community#8290] [Other Feature Feedback, Questions, & Ideas]
If an existing starred repo is added to the list, I need to refresh the page to see it in the List

**Comment by brianlovin** [comment]
Hey @santoshyadavdev — is this still happening for you? We've been fixing a ton of bugs in the last few days, so hopefully this is better now, but let me know if things don't seem right on your end.

**Reply by santoshyadavdev** [reply]
Yes still an issue
![list-feedback](https://user-images.githubusercontent.com/11923975/145684918-f9b3b4ec-ccfc-44d6-92bb-5a15c90ab89c.gif)

**Reply by brianlovin** [reply]
Got it, thanks for the video here — I believe the team is aware of these data consistency quirks, but for now hopefully refreshing the page will sort things out. I'll pass this along 😊
